### PR TITLE
ENH: Add support to PANDAS_GBQ_CREDENTIALS_FILE (#86)

### PR DESCRIPTION
**Pull request by jonathansp**

### CODE REVIEW

File: pandas_gbq/gbq.py [hunk 1]
```diff
@@ -875,7 +881,10 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
         authentication (eg. jupyter iPython notebook on remote host)
     auth_local_webserver : boolean, default False
         Use the [local webserver flow] instead of the [console flow] when
-        getting user credentials.
+        getting user credentials. A file named bigquery_credentials.dat will
+        be created in current dir. You can also set PANDAS_GBQ_CREDENTIALS_FILE
+        environment variable so as to define a specific path to storage this
```
Comment (parthea) [inline pandas_gbq/gbq.py]: storage->store

**Comment by jonathansp** [comment]
Done.

**Comment by jonathansp** [comment]
Thanks, @parthea, we really appreciate your work. Do you have a release plan for this change?

**Comment by andrelu** [comment]
> Thanks, @parthea, we really appreciate your work. Do you have a release plan for this change?
+up

**Comment by parthea** [comment]
@jonathansp I'm hoping to create a new release once #83 is merged (pending approval). It would be great to get your feedback on that PR.

In the interim while we are preparing for the next release, you could install the latest master using:
`pip install https://github.com/pydata/pandas-gbq/archive/master.zip`

**Pull request merged**